codeSnippets/snippets/client-upload-file/src/main/kotlin/com/example/Application.kt

```diff
@@ -21,7 +23,13 @@ fun main() {
                     append(HttpHeaders.ContentDisposition, "filename=ktor_logo.png")
                 })
             }
-        )
+        ) {
+            onUpload { bytesSentTotal, contentLength ->
+                fixedRateTimer("timer", true, 0L, 50L) {
```

Comment:
Why is there a timer here? Will it be created on every upload progress change notification?

Reply:
this is the issue.. thanks, will update

Reply:
removed `fixedRateTimer `  from both examples